tests/Fixtures/Entity/Comment.php

```diff
@@ -18,7 +18,7 @@ class Comment
     private $id;
 
     /**
-     * @ORM\ManyToOne(targetEntity=User::class)
+     * @ORM\ManyToOne(targetEntity=User::class, inversedBy="comments")
```

Comment:
`make migrations-validate` indicated me that there was a problem in the mapping :)